README.md

```diff
@@ -398,6 +398,11 @@ TripDetail tripDetail = amadeus.travel.tripParser.post(body);
 Destination[] directDestinations = amadeus.airport.directDestinations.get(Params
     .with("departureAirportCode","MAD")
     .and("max","2"));
+
+// Flight Availabilites Search POST
+// body can be a String version of your JSON or a JsonObject
+FlightAvailabilitySearch[] flightAvailabilitiesSearch
```

Comment:
@xianqiliu Isn't the resouce name `Flight-availability`? Maybe rename the resource class to `FlightAvailability`?